src/memory/postgresql_store/memory.rs

```diff
@@ -217,16 +247,24 @@ impl<const ADDRESS_LENGTH: usize, const WORD_LENGTH: usize> MemoryADT
         Err(PostgresMemoryError::RetryExhaustedError(MAX_RETRIES))
     }
 }
-
+/// To run the postgresql benchmarks locally, add the following service to your pg_service.conf file (usually under ~/.pg_service.conf):
+/// [cosmian_service]
+/// host=localhost
+/// dbname=cosmian
+/// user=cosmian
+/// password=cosmian
 #[cfg(test)]
 mod tests {
```

Comment:
Module docs is usually written like that (commit this change and run `cargo fmt`):

```suggestion
#[cfg(test)]
mod tests {
//! To run the postgresql benchmarks locally, add the following service to your pg_service.conf file
//! (usually under ~/.pg_service.conf):
//!
//! [cosmian_service]
//! host=localhost
//! dbname=cosmian
//! user=cosmian
//! password=cosmian
```